client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-content/common/hosting-features.tsx

```diff
@@ -1,16 +1,24 @@
 import { code, copy, key, plus } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
 import { BackgroundType9 } from 'calypso/a8c-for-agencies/components/page-section/backgrounds';
 import HostingFeaturesSectionV2 from '../../../common/hosting-features-section-v2';
 
 type Props = {
 	heading: string;
 	isPressable?: boolean;
+	useSignaturePlans?: boolean;
 };
 
-export default function HostingFeatures( { heading, isPressable }: Props ) {
+export default function HostingFeatures( { heading, isPressable, useSignaturePlans }: Props ) {
 	const translate = useTranslate();
 
+	const phpWorkers = useMemo( () => {
+		return useSignaturePlans
+			? translate( '5 PHP workers w/ auto-scaling' )
+			: translate( '10 PHP workers w/ auto-scaling' );
```

Comment:
This creates two translation requests. Can we have just one and pass the count as a parameter?
```
translate(`%(count)s workers w/ auto-scaling', {
    args: {
       count: useSignaturePlans ? 5 : 10
    }
})
```

Reply:
`translate` with parameters does not work in this context because it creates a React node in the array, instead of the expected string.

Maybe using  the `__` and `sprintf`, what do you think?